### PR TITLE
Remove software dependency handling, remove historical artifacts

### PR DESCRIPTION
**Pull request by pinin4fjords**
This PR:

 * Moves Galaxy parameters file to YAML format preferred by newly packaged galaxy-workflow-executor.
 * Removes workflow executor submodule- we will now assume this software is installed in the calling environment.
 * Removes conda environment handling for same reason
 * Removes processes associated with TPM filtering now done as part of https://github.com/ebi-gene-expression-group/scxa-bundle-workflow
 * Removes things associated with bundles, which date from a time when bundles existed before this process and the overall control workflow was very different. 

Paired with https://github.com/ebi-gene-expression-group/scxa-control-workflow/pull/17

### CODE REVIEW

File: w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml [hunk 1]
```diff
@@ -0,0 +1,246 @@
+barcodes: {}
```
Comment (pcm32) [inline w_droplet_clustering/scanpy_clustering_workflow_parameters.yaml:1]: You might want to delete these two (barcodes, cellmeta) if they are inputs.

File: w_smart-seq_clustering/scanpy_clustering_workflow_parameters.yaml [hunk 2]
```diff
@@ -0,0 +1,246 @@
+barcodes: {}
```
Comment (pcm32) [inline w_smart-seq_clustering/scanpy_clustering_workflow_parameters.yaml:1]: remove if inputs (unless that you have tried this and all works well).

**Review by pcm32 — APPROVED** [review]
LGTM, besides some comments in parameters. But if you have tried this and it is working well, go ahead.

File: w_smart-seq_clustering/scanpy_clustering_workflow_parameters.yaml (hunk 2 above)
Comment (pinin4fjords) [reply w_smart-seq_clustering/scanpy_clustering_workflow_parameters.yaml:1]: It works fine (we've been doing it all along with the json)

**Comment by pinin4fjords** [comment]
Thanks very much @pcm32 !

**Pull request merged**